### ISSUE TITLE
Change Namespace

### ISSUE THREAD

**Issue by cord** [pdmfc/nova-info-card#7]
nice package!

There seems to be an issue with the namespace, I need to write:

```
use Pdmfc\InfoCard\InfoCard;
(...)

(new InfoCard())
                    ->info('...')

```

to make it work

**Comment by skoyah** [comment]
Thanks @cord! I'll take a look tomorrow to fix it!

**Comment by skoyah** [comment]
So after some thinking I've decided to leave it as it is for now since is a not a bug. The problem is in the README itself. I'll update the docs and will target this issue for the 1.0 milestone.

Changing the namespace is a breaking change, but you are right.  It is a bit messy the current one. Will change it to something better when the 1.0 comes out.

Thanks once again @cord!

**Comment by skoyah** [comment]
Resolved a12e79b.

**Comment by skoyah** [comment]
Hey @cord  just letting you know we've bumped up the package version to 1.0 with the updated namespace!
